tools/config.ts

```diff
@@ -17,50 +17,78 @@ export const APP_SRC              = 'app';
 export const ASSETS_SRC           = `${APP_SRC}/assets`;
 
 export const TOOLS_DIR            = 'tools';
+export const TMP_DIR              = 'tmp';
 export const TEST_DEST            = 'test';
 export const DOCS_DEST            = 'docs';
 export const APP_DEST             = `dist/${ENV}`;
 export const ASSETS_DEST          = `${APP_DEST}/assets`;
+export const BUNDLES_DEST         = `${APP_DEST}/bundles`;
 export const CSS_DEST             = `${APP_DEST}/css`;
 export const FONTS_DEST           = `${APP_DEST}/fonts`;
 export const LIB_DEST             = `${APP_DEST}/lib`;
+export const APP_ROOT             = ENV === 'dev' ? `${APP_BASE}${APP_DEST}/` : `${APP_BASE}`;
 export const VERSION              = appVersion();
 
 export const VERSION_NPM          = '3.0.0';
 export const VERSION_NODE         = '4.0.0';
 
 // Declare NPM dependencies (Note that globs should not be injected).
-export const DEV_DEPENDENCIES = [
-  { src: 'systemjs/dist/system-polyfills.js' },
+export const NPM_DEPENDENCIES = [
+  { src: 'systemjs/dist/system-polyfills.js', dest: LIB_DEST },
```

Comment:
Can't we get rid of `LIB_DEST`? We may declare it in production config, something like:

```ts
const PROD_CONFIG = {
  CSS_DEST,
  LIB_DEST
};
```